Mathlib/LinearAlgebra/Finsupp/Pi.lean

```diff
@@ -145,3 +145,78 @@ theorem range_mapRange_linearMap (f : M →ₗ[R] N) (hf : LinearMap.ker f = ⊥
     <;> simp_all (config := {contextual := true}) [← hy, map_zero, LinearMap.ker_eq_bot'.1 hf]
 
 end Finsupp
+
+namespace FunOnFinite
```

Comment:
Do you mind adding the main defs / results in this section to the module docstring?